brdnet/evaluate_models.py

```diff
@@ -96,7 +114,7 @@ def eval_pyod_model(model_name, train_X, train_Y, val_X, val_Y):
     module = importlib.import_module(model_module)
     model = getattr(module, model_name)
 
-    model_instance = model()
+    model_instance = model(contamination=.5)
```

Comment:
Outlier detection methods assume that there are two classes of data, normal data and outliers. When training the methods, the contamination parameter is an estimate of how much of your training data is outliers. Since this problem is a fairly balanced two class classification problem I set the `contamination` parameter to .5 instead of the default .1